Update Building-in-Mac-OSX.md (#506)
betaflight/betaflight.com: docs/development/building/Building-in-Mac-OSX.md

## Build the code

Enter the betaflight directory and run `make configs` to retrieve board targets then `make MATEKH743` to

> build firmware for the BETAFLIGHTF4. When the build completes, the .hex firmware should be available as

Reviewer: this line still reports `BETAFLIGHTF4`

> `obj/betaflight_4.4.2_MATEKH743.hex` for you to flash using the Betaflight Configurator.

Reviewer 2: ```suggestion
`obj/betaflight_4.6.0_MATEKH743.hex` for you to flash using the Betaflight Configurator.
```

## Build the code

Enter the betaflight directory and run `make configs` to retrieve board targets then `make MATEKH743` to

> build firmware for the BETAFLIGHTF4. When the build completes, the .hex firmware should be available as

Reviewer 2: ```suggestion
build firmware for the MATEKH743. When the build completes, the .hex firmware should be available as
```

git reset --hard
git pull

> make clean

Reviewer 3: make CONFIG=MATEKH743

Reviewer 2: - `rm -rf obj` is less  typing :)
- can we make this `make clean MATEKH743` and `make clean` for all?

Reviewer: we have `make MATEKH743_clean` as well, so what id preferred? (or no preference?)

Reviewer 2: `make MATEKH743_clean` is ugly :)

> make clean

Reviewer 2: ```suggestion
make clean CONFIG=MATEKH743
```
